configs/ASRock/Z390-Extreme4.conf

```diff
@@ -0,0 +1,107 @@
+# ASRock Z3900 Extreme4
+# 2019, contributed by Alex Cherkayev <doctor64@gmail.com>
+#
+# dmi: board_name:   Z390 Extreme4
+# dmi: board_vendor: ASRock
+# dmi: bios_version: P4.00
+# cpu: Intel(R) Core(TM) i7-9700K CPU
+chip "nct6791-isa-*"
+
+    # Fans
+    label fan1 "CHA Fan 3"
+    label fan2 "CPU Fan 1"
+    label fan3 "CPU Fan 2/WP"
+    label fan4 "CHA Fan 1"
+    label fan5 "CHA Fan 2"
+
+    # Temperatures
+    label temp1 "Motherboard"
+    set temp1_max 55
+    set temp1_max_hyst 50
+    label temp2 "CPU MB"
+    set temp2_max 100
+    set temp2_max_hyst 75
+
+    # temp3 always 22
+    # temp4 102 in idle, 103 under load
+    # temp5 always 15
+    # temp6 always 12 
+    ignore temp3
+    ignore temp4
+    ignore temp5
+    ignore temp6
+    
+    #PECI Agent 0 match CPU cores temp    
+    label temp7 "CPU Core"
+    
+    # PCH_CHIP_CPU_MAX_TEMP PCH_CHIP_TEMP PCH_CPU_TEMP always 0.0C, ignore
+    ignore temp8
+    ignore temp9
+    ignore temp10
+
+    # Voltages
+    
+    label in0 "VCore Voltage"
+    compute in0 @*2, @/2
+    set in0_min 0.40 
+    set in0_max 1.40
+
+    label in1 "+5V"
+    # deducted, not confirmed
+    compute in1 ((20/10)+1)*@, @/((20/10)+1)
+    set in1_min 5 * 0.95
+    set in1_max 5 * 1.05
+
+    label in4 "+12V" 
+    # deducted, not confirmed
+    compute in4 @*12, @/12
+    set in4_min 12 * 0.95
+    set in4_max 12 * 1.05
+
+    #unknown, always 1.18V just set limits to remove alarm
+    set in5_min 1.1
+    set in5_max 1.3
+
+    label in6 "PCH +1.0V"
+    # limit taken from intel 300 series pch datasheet
+    # set in6_min 0.99
+    # set in6_max 1.10 
+    # verifyed by changing in bios, limit taken from bios
```

Comment:
Spelling: verified. Multiple occurences.

Reply:
Sorry, not fully understand what's wrong here. First set of limits taken from intel datasheet, second, taken from actual BIOS

Reply:
Ahh, "verify" but "verified". Sorry, English language not my strongest side :)

Reply:
No worries :).

Reply:
fixed